### DISCUSSION TITLE
Installation i system context

### DISCUSSION THREAD

**Discussion by Michael-HubGit** [Eduap-com/WordMat#184] [Support]
Hej,
Vi benytter SCCM til at installere software på computerne. Dette er bl.a. på grund af at brugerne ikke er administrator på deres computere.
Desværre oplever vi en forskellighed på hvordan WordMat opføre sig. Når jeg installere det via .cmd script i hånden, kontra når SCCM installere via det samme script.

Jeg har et installationsscript der installere WordMat med følgende parameter.
"%~dp0WordMat128-1.exe" /LOADINF="%~dp0WordMat128-1.inf" /SP /VERYSILENT /SUPPRESSMSGBOXES /NORESTART

Hvis jeg selv installere det "manuelt" ved at højreklikke på .cmd scriptet, og vælger "_Kør som Administrator_". Så installere WordMat fint, fanen kommer frem i Word, og WordMat kan benyttes.

Hvis jeg sætter SCCM til at installere WordMat, med det samme script. Så bliver WordMat også installeret og fanen kommer frem i Word.
MEN. Så bliver brugeren mødt af en **SIKKERHEDSVARSEL**, hvor man bliver bedt om at aktivere makros.
![WordMat Makro advarsel](https://github.com/Eduap-com/WordMat/assets/159438371/71eecd0c-f254-4b1c-80d9-994c12692f29)

Hvorfor sker dette? Hvorfor bliver brugeren kun mødt af makro advarsel, når WordMat bliver installeret i System Context.
Og hvordan kommer vi uden om det?

Brugerne er ikke administrator på computerne, så de har ikke rettigheder til at lave nogen ændringer på computerne.

**Comment by Stygges** [comment]
Hej Michael 

Jeg havde en del problemer med det her også, og jeg tror jeg har et svar til dig, selvom dem måske ikke er særlig brugbart. 

In version 1,28 og1,29 wordmat bliver wordmat ikke længer installeret med system context. 

Du kan teste dette ved at gøre følgende.

PsExec (https://learn.microsoft.com/en-us/sysinternals/downloads/psexec) tillader at du kan køre Command Prompt som SYSTEM. 
 
Run the following steps:
1.	kør Command Prompt som Administrator
2.	kør PsExec med -s -i cmd.exe parameter (i command prompt)
3.	kør Wordmat installer (med dine arguments/parameter) 

Så burde du kunne teste om versionen af wordmat kan klare dine krav. 

Du er enten nød til at lave en generel indstilling i word, eller vente på en version der klare det igen. ... eller bruge 1,26 som understøtter det. 

Håber det hjælper lidt

Lars

**Comment by InterceptorDK** [comment]
Vi har samme problem, Jeg har ver. 27 uden, men kan ikke installerer nyere. Også via System. Ser ud til en bevist ændring, fik at vide at hvis jeg købte support, kan de løse det. Så det er nok ikke noget der bliver fixet.
